modules/client/src/main/scala/ch/epfl/bluebrain/nexus/admin/client/AdminClient.scala

```diff
@@ -40,9 +41,12 @@ trait AdminClient[F[_]] {
     * Retrieves ACLs for a given project resource instance.
     *
     * @param name        the project name
+    * @param parents     matches only the provided ''path'' (false) or the parents also (true)
+    * @param self        matches only the provided ''identities'' (true) or any identity which has the right own access (true)
```

Comment:
matches only the provided ''identities'' (true) or any identity which has the right own access (false)